### ISSUE TITLE
jsx-no-undef doesn't catch object access

### ISSUE THREAD

**Issue by jdiaz5513** [yannickcr/eslint-plugin-react#172]
This warns as expected:

```javascript
import Button from 'button';

let b = <Bottom>oops</Bottom>;
```

This does not:

```javascript
import Buttons from 'buttons';

let b = <Bottoms.BigButton>oops</Bottoms.BigButton>
```

It should be telling me that `Bottoms` is not defined in this scope.
